rmf_site_format/src/sdf.rs

```diff
@@ -434,6 +434,11 @@ impl Site {
             filename: "toggle_floors".into(),
             ..Default::default()
         };
+        // Only export default scenario into SDF for now
+        let (_, default_scenario) = self
+            .scenarios
+            .first_key_value()
+            .expect("No scenarios found");
```

Comment:
[a2e2ef5](https://github.com/open-rmf/rmf_site/pull/242/commits/a2e2ef5061c6d2ea64a51136c6cf97dafda11c52)

I was wondering if it is necessary to also remove the `.expect()` found here. I took a stab at it but could not find an elegant way to raise errors without making multiple loops over each vector, mainly because each element is a tuple (member variables are `Vec<(T, Pose)>`), so I had trouble converting them into `Result<Vec<(T, Pose)>>` with `Pose` in the mix.

Reply:
> I was wondering if it is necessary to also remove the .expect() found here.

Did you mean to link to somewhere else in the code? I see that you've removed the `.expect(~)` that my inline comment was referring to, so I don't think your question is related to this one.

Reply:
Ah I didn't realize I missed out on attaching the link, but I think you found what I was referring to. Thanks!